package.json

```diff
@@ -120,6 +120,7 @@
     "express": "^4.17.1",
     "express-list-endpoints": "^5.0.0",
     "express-winston": "^4.1.0",
+    "fs-reverse": "0.0.3",
```

Comment:
What do you think about vendoring this dep? It seems no longer maintained, and repo owner archived it. If we have to define the types anyway, we could just pull in the code and use ts directly? I didn't find any good alternative libs in a cursory search.